papers/michael_lange/michael_lange.rst

```diff
@@ -74,14 +78,19 @@ with existing open-source software.
 
 While Devito was originally developed for seismic imaging workflows,
 the automated generation and optimization of stencil codes can be
-utilised for a much broader set of computational problems. In this
-paper we will give a brief overview of the design concepts and
-Devito's key features before demonstrating the Devito API on a set of
-classic examples from computational fluid dynamics (CFD). Based on
-this, we will then highlight the use of Devito in an example of a
-complex seismic inversion algorithms to demonstrate its use in
-practical scientific applications and to showcase the performance
-achieved by the auto-generated and optimised code.
+utilised for a much broader set of computational problems. Matrix-free
+stencil operators based on explicit finite difference schemes are
+widely used in industrial and academic research, although they merely
+represent one of many approaches to solving PDEs [Baba16]_, [Liu09]_,
+[Rai91]_. In this paper we therefore limit our discussion of numerical
+methods to a minium and instead focus on the ease which these
+operators can be created symbolically. We give a brief overview of the
+design concepts and Devito's key features and demonstrate the Devito
+API using a set of classic examples from computational fluid dynamics
+(CFD). Based on this, we will then highlight the use of Devito in an
+example of a complex seismic inversion algorithms to demonstrate its
```

Comment:
"demonstrate" is used many times before as well as after this instance. Maybe use "show" ?